OracleDatabase/SingleInstance/extensions/sharding/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Database with Fast Fail Over support
+#
+# REQUIREMETNS FOR THIS IMAGE
+# ----------------------------------
+# Any release of prebuilt oracle/database base docker image
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+#
+# Run:
+#      $ docker build -t <extended_image_name> . --build-arg BASE_IMAGE=18.3.0-ee
+#
+
+ARG BASE_IMAGE_VERSION=19.3.0
+ARG BASE_IMAGE=oracle/database:${BASE_IMAGE_VERSION}-ee
+ARG BASE_REPO=https://github.com/oracle/db-sharding
+
+# Creating stage for downloading necessary script files
+FROM ${BASE_IMAGE} as downloader
```

Comment:
Can this first stage be from oracle-linux slim image

Reply:
I checked with oraclelinux:7-slim image, it does not have 'tar' or 'unzip' utilities. So, we can't use it in the first stage.

Reply:
You could just install them via `yum` if that would make things easier/quicker.

Reply:
Thanks @gvenzl , now oraclelinux:7-slim is used in downloader stage. We only need this stage for downloading the archived repo and extract files, oraclelinux image will make things cleaner.

Reply:
Yeah, I thought so. :)